src/lib/seam/components/AccessCodeTable/AccessCodeHealthBar.tsx

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash'
```

Comment:
@dawnho Please do not add new libs to this project. We are limiting deps. You can merge natively with `[...erroredAccessCodes, ...warningAccessCodes]`. :)

Reply:
there will be duplicates if i do it this way, but i'll find another way